packages/@aws-cdk/aws-codecommit/lib/repository.ts

```diff
@@ -53,6 +55,14 @@ export abstract class RepositoryRef extends cdk.Construct {
         };
     }
 
+    public addToPipelineStage(stage: actions.IStage, name: string, props: CommonPipelineSourceProps): PipelineSource {
```

Comment:
The word `Stage` is redundant (since the type of the first argument tells this story). `addToPipeline` should be fine in my mind

Reply:
Hah, that's funny. `addToPipeline` is actually the name I proposed originally, and @RomainMuller asked to change it to `addToPipelineStage` :)